gallery/visualtests/nxTag.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { clickTest, focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+
+describe('NxTag', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxTag');
+  });
+
+  describe('Basic NxTag', function() {
+    const selector = '#nx-tag-example .test-nx-tag';
+
+    it('looks right', simpleTest(selector));
+  });
+
+
+  describe('NxTag Selectable', function() {
+    const selector = '#nx-selectable-tag-example .nx-tag:first-child';
+
+    it('has a grey border and grey background by default', simpleTest(selector));
+    it('has a dark grey border when hovered', hoverTest(selector));
+    it('has a light blue glow and light blue border when focused', focusTest(selector));
+    it('has a light blue glow and dark grey border when focused and hovered', focusAndHoverTest(selector));
+    it('has a dark grey border and dark blue background when clicked', clickTest(selector));
```

Comment:
`clickTest` tests what it looks like when the mouse is depressed on it.  I think what we more want to test is what it looks like after it has been clicked/selected.

Reply:
> `clickTest` tests what it looks like when the mouse is depressed on it. I think what we more want to test is what it looks like after it has been clicked/selected.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/50a30d99b619217f0ae5d449f0fc3a600dba8fdb